app/helpers/application_helper.rb

```diff
@@ -1,2 +1,9 @@
 module ApplicationHelper
+  def user_category_options
+    [
+      ["manager", "Manager"],
+      ["developer", "Developer"],
+      ["qa", "QA"],
+    ]
```

Comment:
```suggestion
    User::USER_CATEGORIES.keys.map { |x| [x.to_s, x.to_s.titleize] }
```

That way you don't need to update the array every time you make a change in the `USER_CATEGORIES`